extensions/parquet/compression/build.gradle

```diff
@@ -10,11 +10,14 @@ dependencies {
     Classpaths.inheritParquetHadoopConfiguration(project)
 
     implementation project(':Configuration')
+    implementation project(':util-channel')
```

Comment:
This is *just* for `Resource` and `SeekableChannelContext`, right? I wonder if the interface for SCC could be simplified to something like `<RESOURCE_TYPE extends AutoCloseable> RESOURCE_TYPE getResource(Supplier<RESOURCE_TYPE>)`, because then we could just have the input to `decompress` be `Function<Supplier<RESOURCE_TYPE>, RESOURCE_TYPE>` and get rid of the dependency.